### PR TITLE
[DOC] Enhanced RDoc for Net::HTTP

### PR DESCRIPTION
**Pull request by BurdetteLamar**
Changes to second half of attributes.  This is mostly additions rather than revisions.

### CODE REVIEW

File: lib/net/http.rb [hunk 1]
```diff
@@ -1211,63 +1211,75 @@ def use_ssl=(flag)
       :verify_hostname,
     ] # :nodoc:
 
-    # Sets path of a CA certification file in PEM format.
-    #
-    # The file can contain several CA certificates.
+    # Sets or returns the path to a CA certification file in PEM format.
+    # See {OpenSSL}[https://docs.ruby-lang.org/en/master/OpenSSL.html]
```
[the remark this answers is not in the document]
Comment (BurdetteLamar) [reply lib/net/http.rb:1215]: Referring to just this one, or to all such?

[the remark this answers is not in the document]
Comment (peterzhu2118) [reply lib/net/http.rb:1215]: All of the ones that just refer to `OpenSSL` (and not a specific method).

[the remark this answers is not in the document]
Comment (BurdetteLamar) [reply lib/net/http.rb]: Removed (10 places).

**Comment by BurdetteLamar** [comment]
@peterzhu2118, ready for you.

File: lib/net/http.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
[the remark this answers is not in the document]
Comment (BurdetteLamar) [reply lib/net/http.rb]: Removed.

File: lib/net/http.rb [hunk 3]
```diff
@@ -1497,17 +1500,20 @@ def proxy_class?
...
 
-      # User name for accessing proxy. If \Net::HTTP does not use a proxy, nil.
+      # Returns the user name for accessing the proxy, or +nil+ if none;
+      # see -Net::HTTP@Proxy+Server.
       attr_reader :proxy_user
 
-      # User password for accessing proxy. \If Net::HTTP does not use a proxy,
-      # nil.
+      # Returns the password for accessing the proxy, or +nil+ if none;
+      # see -Net::HTTP@Proxy+Server.
```
[the remark this answers is not in the document]
Comment (BurdetteLamar) [reply lib/net/http.rb]: Removed (3 places).

File: lib/net/http.rb [hunk 4]
```diff
@@ -1211,62 +1212,64 @@ def use_ssl=(flag)
...
-    # Sets to check the server certificate is valid for the hostname.
-    # See OpenSSL::SSL::SSLContext#verify_hostname=
+    # Sets or returns whether to verify that the server certificate is valid
+    # for the hostname.
+    # See {OpenSSL::SSL::SSLContext#verify_hostname=}[https://docs.ruby-lang.org/en/master/OpenSSL/SSL/SSLContext.html#attribute-i-verify_mode].
     attr_accessor :verify_hostname
 
-    # The X509 certificate chain (an array of strings) for the session's socket peer,
+    # Returns the X509 certificate chain (an array of strings)
+    # for the session's socket peer,
```
[the remark this answers is not in the document]
Comment (BurdetteLamar) [reply lib/net/http.rb:1271]: Sentence continues on next line;  so stet?

[the remark this answers is not in the document]
Comment (peterzhu2118) [reply lib/net/http.rb:1271]: Oh oops I didn't notice that.

**Review by peterzhu2118 — APPROVED** [review]

**Pull request merged**